### ISSUE TITLE
Remove example

### ISSUE THREAD

**Issue by brandwe** [AzureAD/passport-azure-ad#119]
Example is old and not maintained. Should either fix or remove. Our other libs do not have samples in SDK, just tests.

**Comment by lovemaths** [comment]
We should at least have a branch keeping (and fixing) these examples for our development testing purposes. Here we have 6 examples for different strategies, but azure sample website just has 1 example, which is for OIDC strategy.

**Comment by dhodgin** [comment]
I've been working with AzureAD and learning how to integrate it into a MEAN stack application. I've found that all of the different examples out there are both outdated and more importantly not very well documented. Especially the different config options that the libraries use. Sometimes tenant is tenantId, sometimes its realm, sometimes its audience. some options are just flat out never explained unless you happen to find out on some other repo for a completely different stack.

It would be a HUGE help for end users to have a really good example SPA (single page app) end to end using angular and node with adal-angular on client side and passport-azure-ad on the server side and to properly document the example and how it works and what the config values represent.

In my opinion after spending over 4 weeks trying to just get a sample sandbox app working end to end this time could have been shortened to a week easily if everything was documented and explained with the sample code. i've gone through at least 20 different browser tabs with bits and pieces of info gleaned from other examples and some of the azure high level documentation to finally put all the pieces together.

I was planning on just putting up my sample sandbox app that uses angular, express, and node without mongo to minimize the amount of code and complexity. I doubt this repo is the right place for such an example app because really this is just about server side passport authentication wrappers but the examples in this repo were still very helpful and without them I would have probably not even tried to integrate our SaaS with AAD. (Which probably would get me fired as we are working closely with Microsoft on some business opportunities and they are pushing for AAD SSO integration)

A lot of the comments mix BearerStrategy and OIDCStrategy up too by the way. I didn't even try to get the OIDCStrategy example working yet because there are 9 config options and half of them I have no idea if they are required or what they should be set to. I did manage to get the BearerStrategy working in a sample app but have a long way to go still before it will make it into our production environment.
